src/beautifiers/index.coffee

```diff
@@ -325,6 +325,12 @@ module.exports = class Beautifiers extends EventEmitter
         languages = @languages.getLanguages({grammar, extension: fileExtension})
         logger.verbose(languages, grammar, fileExtension)
 
+        # use fallback if configured
+        unsupportedLanguageFallback = atom.config.get("atom-beautify.unsupportedLanguageFallback")
+        if languages.length < 1 && unsupportedLanguageFallback.charAt(0) != '-'
+          languages = [require('../languages/' + unsupportedLanguageFallback)]
```

Comment:
This should be changed to something like:

```coffee
languages = @languages.getLanguages({name: unsupportedLanguageFallback})
```

See https://github.com/Glavin001/atom-beautify/blob/master/src/languages/index.coffee#L82 for details.